index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body>
+ <div id="main"> 
+  <nav>
+    <div>
+      <h1 id="title">Bjarne Stroustrup</h1>
+      <h3>He is the creator of C++ programming language</h3>
+    </div>
+  </nav>
+  <section class="images-section">
+    <div class="images" id="img-div">
+      <img id="image" src="images/Bjarne_Stroustrup_UC3M_2-1.jpg" alt="">
+      <p id="img-caption">Stroustrup was born in Aarhus, Denmark. His family was working class, and he went to the local schools.[7] He attended Aarhus University 1969–1975 and graduated with a master's degree in mathematics and computer science. His interests focused on microprogramming and machine architecture. He learned the fundamentals of object-oriented programming from its inventor, Kristen Nygaard, who frequently visited Aarhus.</p>
+    </div>
+  </section>
+  <section id="tribute-info">
+    <div>
+      <h1>Here's a time line of Bjarne Stroustrup:</h1>
```

Comment:
For good coding practice, you need to make sure there is only one H1, so kindly modify your code.